lms/djangoapps/commerce/api/v1/serializers.py

```diff
@@ -26,11 +29,32 @@ class CourseSerializer(serializers.Serializer):
     """ Course serializer. """
     id = serializers.CharField()  # pylint: disable=invalid-name
     name = serializers.CharField(read_only=True)
+    verification_deadline = serializers.DateTimeField(blank=True)
     modes = CourseModeSerializer(many=True, allow_add_remove=True)
 
+    def validate(self, attrs):
```

Comment:
Updated per my conversation with @wedaly. Also added more comments to better explain what we are doing here.